sycl/test-e2e/KernelFusion/lit.local.cfg

```diff
@@ -0,0 +1,5 @@
+if config.sycl_be in ['ext_oneapi_hip']:
+   config.unsupported = True
+
+if 'fusion' not in config.available_features:
+   config.unsupported = True
```

Comment:
I may be missing some context from the previous PRs: Does this mean none of the tests in the `KernelFusion` folder run, if the HIP backend is specified, even if other backends are specified? 
So, for example, with `opencl:cpu,gpu;ext_oneapi_hip:gpu`, would the tests still execute on OpenCL?

Reply:
Ok, thanks for the explanation.